kubernetes/cleanup

```diff
@@ -2,5 +2,11 @@
 
 log()   { echo ">>> $1" ; }
 
+[[ -d manifest-templates ]] || { echo >&2 "Please run this script from within the kubernetes folder"; exit 1; }
```

Comment:
I've ran this script standalone several times. I was following the same patterns as the start script here. 😀